test/e2e/egress_test.go

```diff
@@ -38,7 +38,7 @@ import (
 	controllerconfig "antrea.io/antrea/pkg/config/controller"
 )
 
-const waitEgressRealizedTimeout = 3 * time.Second
+const waitEgressRealizedTimeout = 15 * time.Second
```

Comment:
Would membership take 10+ seconds to detect the failed node?

Reply:
During the tests, I found that failure detection usually takes 6~9 seconds. I think 15 seconds is enough to avoid test flakies. Do you think we should decrease it to some value like 10 or 12 seconds?

Reply:
OK, if it usually takes 6-9s, 15s makes sense as timeout to make the test stable.
I thought it takes only 3s to failover. Maybe let's take a look at how is the time spent and try to optimize it later.